src/test/fixtures/config/credential.tsx

```diff
@@ -0,0 +1,77 @@
+import { v2 } from "@govtechsg/open-attestation";
+import { Credential, DidCredential } from "./types";
+
+export const ropstenCredential: Credential = {
+  network: "ropsten",
+  wallet: {
+    type: "ENCRYPTED_JSON",
+    encryptedJson:
+      '{"address":"1245e5b64d785b25057f7438f715f4aa5d965733","id":"bf069d1b-4e88-487c-b695-f2e03ed7c1ff","version":3,"Crypto":{"cipher":"aes-128-ctr","cipherparams":{"iv":"1f34e7bfdaee4b0778ecea4c8d12a543"},"ciphertext":"4f6cee88b4776f4f6f8eedf3da11c6a13542aa4bb65d46a5c1bc44c100a96f54","kdf":"scrypt","kdfparams":{"salt":"d931e0ea13032fd70060e40054c5a76c0571f4d840ec91eeda1bf68bdcad84db","n":1,"dklen":32,"p":1,"r":8},"mac":"06c7897e3ff04245bf4f0765d8b6a8482c1c9981cb46ae88f636f9c83cd0b891"},"x-ethers":{"client":"ethers.js","gethFilename":"UTC--2020-05-15T09-03-13.0Z--1245e5b64d785b25057f7438f715f4aa5d965733","mnemonicCounter":"99b7f5b6897dcfe22fc7aa00d8e3cf5e","mnemonicCiphertext":"6e7c1d38f162e54050b125f1f51b43ca","path":"m/44\'/60\'/0\'/0/0","version":"0.1"}}',
+  },
+  tokenRegistry: "0x72d9a82203Ef9177239A5E3cB7A8FB9a78D04f17",
+  documentStore: "0x8bA63EAB43342AAc3AdBB4B827b68Cf4aAE5Caca",
+  identityProof: "demo-tradetrust.openattestation.com",
+  documentStorage: {
+    apiKey: "randomKey",
+    url: "https://api-ropsten.tradetrust.io/storage",
+  },
+};
+
+export const ropstenAWSCredential: Credential = {
+  network: "ropsten",
+  wallet: {
+    type: "AWS_KMS",
+    accessKeyId: "<access key id>",
+    region: "<region>",
+    kmsKeyId: "<kms key id>",
+  },
+  tokenRegistry: "",
+  documentStore: "",
+  identityProof: "",
+  documentStorage: {
+    apiKey: "randomKey",
+    url: "https://api-ropsten.tradetrust.io/storage",
+  },
+};
+
+export const rinkebyCredential: Credential = {
+  network: "rinkeby",
+  wallet: {
+    type: "ENCRYPTED_JSON",
+    encryptedJson:
+      '{"address":"1245e5b64d785b25057f7438f715f4aa5d965733","id":"bf069d1b-4e88-487c-b695-f2e03ed7c1ff","version":3,"Crypto":{"cipher":"aes-128-ctr","cipherparams":{"iv":"1f34e7bfdaee4b0778ecea4c8d12a543"},"ciphertext":"4f6cee88b4776f4f6f8eedf3da11c6a13542aa4bb65d46a5c1bc44c100a96f54","kdf":"scrypt","kdfparams":{"salt":"d931e0ea13032fd70060e40054c5a76c0571f4d840ec91eeda1bf68bdcad84db","n":1,"dklen":32,"p":1,"r":8},"mac":"06c7897e3ff04245bf4f0765d8b6a8482c1c9981cb46ae88f636f9c83cd0b891"},"x-ethers":{"client":"ethers.js","gethFilename":"UTC--2020-05-15T09-03-13.0Z--1245e5b64d785b25057f7438f715f4aa5d965733","mnemonicCounter":"99b7f5b6897dcfe22fc7aa00d8e3cf5e","mnemonicCiphertext":"6e7c1d38f162e54050b125f1f51b43ca","path":"m/44\'/60\'/0\'/0/0","version":"0.1"}}',
+  },
+  tokenRegistry: "0x26E730520949F9B2F73b53A35044680c2165725D",
+  documentStore: "0x8bA63EAB43342AAc3AdBB4B827b68Cf4aAE5Caca",
+  identityProof: "demo-tradetrust.openattestation.com",
+  documentStorage: {
+    apiKey: "randomKey",
+    url: "https://api-rinkeby.tradetrust.io/storage",
+  },
+};
+
+export const didCredential: DidCredential = {
+  network: "homestead",
```

Comment:
leave a comment on DID not necessary has to depend on network? or is it ok to omit this entirely?

Reply:
sadly we can't omit this cause the validator check if there is network and within the acceptance enum of "homestead, local, ropsten, rinkeby". i tried local but it has different processing flow thus failed